R/source.R

```diff
@@ -22,11 +22,26 @@ source_package <- function(path) {
   load_pkg_dependencies(path)
 
   paths <- package_files(path)
-  lapply(paths, sys.source, envir = env, keep.source = FALSE)
+  lapply(paths, sys_source, envir = env)
 
   env
 }
 
+sys_source <- function(file, envir = baseenv()) {
```

Comment:
My bad, not notice `local` accepts `environment` values. So just use ` source(file, encoding = "UTF-8", keep.source = FALSE, local = envir)` is ok then.